client/src/components/Reviews/Reviews.jsx

```diff
@@ -8,32 +7,36 @@ class Reviews extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      hello: 'world',
-      reviews: []
+      reviews: [],
     };
   }
+
+  componentDidMount() {
+    const paths = window.location.pathname.substr(1).split('/');
+    const roomNum = paths[1];
+    this.getAllReviews(roomNum);
+  }
+
   getAllReviews(roomId) {
     fetch(`/api/rooms/${roomId}/reviews`)
       .then(res => res.json())
-      .then(reviews => {
-        console.log(reviews)
-        return this.setState({reviews: reviews})
+      .then((reviews) => {
+        // reviews.reduce((review) => {
+        //   this.setState();
+        // });
+        console.log(reviews);
+        return this.setState({ reviews });
```

Comment:
Since you have only one property in the state, this probably works without any issue. However, it may be safer to call setState as {reviews: reviews}